cms/models_test.py

```diff
@@ -870,6 +877,34 @@ def test_external_course_page_properties():
     assert external_course_page.product == external_course_page.course
 
 
+def test_course_page_format_field_default_value():
+    """
+    Verifies that the "format" field in a course page has the default value FORMAT_ONLINE.
+    """
+    course_page = CoursePageFactory.create()
+
+    assert course_page.format == FORMAT_ONLINE
+
+
+@pytest.mark.parametrize("format_value", [FORMAT_ONLINE, FORMAT_OTHER])
+def test_course_page_format_field_values(format_value):
+    """
+    Verifies that the "format" field in a course page has the default value FORMAT_ONLINE
+    and same context.
+    """
+    course_page = CoursePageFactory.create(format=format_value)
+    staff_user = StaffUserFactory.create()

```

Comment:
You can use `staff_user` fixture defined [here](https://github.com/mitodl/mitxpro/blob/master/fixtures/common.py#L20).

Reply:
Addressed.